app/subscriber/src/features/settings/MyMinisterSettings.tsx

```diff
@@ -84,7 +88,7 @@ export const MyMinisterSettings: React.FC = () => {
                   }
                 }}
               />
-              <div className="aliases">Search Aliases: {o.aliases}</div>
+              <div className="position">{o.position}</div>
```

Comment:
Was this requested?  I don't think we have the `position` data presently, which would result in this being blank.

Reply:
Yah we have `position` data now.  Carolynn has been entering it.

When you are back next week we can talk about the `description` field on the `ministers` table.  It doesn't have a purpose currently, so could arguably be hidden in the Editor UI.
Also, the Organization table/relationship you added before you left isn't quite that straight forward apparently.  Carolynn showed me that there are cases where a minister can have multiple parent orgs :(

Reply:
They don't have any of this functionality in the existing product, so we should limit how much we add for MVP.